src/Tools/Source/RunTests/Options.cs

```diff
@@ -176,7 +176,7 @@ bool isOption(string argument, string optionName, out string value)
                     opt.ProcDumpDirectory = value;
                     index++;
                 }
-                else if (comparer.Equals(current, "-procdump"))
+                else if (comparer.Equals(current, "-useprocdump"))
                 {
                     opt.UseProcDump = false;
```

Comment:
@jaredpar should this be `true`?

Reply:
Yes. That is fixed in another branch that will merge into this one soon.